AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -234,42 +229,9 @@ bool GCS_MAVLINK_Tracker::try_send_message(enum ap_message id)
         CHECK_PAYLOAD_SIZE(HWSTATUS);
         tracker.send_hwstatus(chan);
```

Comment:
Maybe I'm just nitpicking, but is there any reason you can see why Tracker wouldn't do the same as the others? I understand you didn't move this one because it actually sends this message but with zeros - I'm wondering what's useful in that and why can't we send the actual board voltage?